src/bindings/python/docs/code_examples.md

```diff
@@ -0,0 +1,413 @@
+# Examples of OpenVINO:tm: Python API code
+<!-- Do we keep it as is? -->
+
+#### Prerequisites
+<!-- Do we keep it here? -->
+*To be added...*
+
+##### Enviroment
+<!-- Link to enviroment setup -->
+*To be added...*
+
+##### Building
+<!-- Link to building instructions -->
+*To be added...*
+
+### Different ways of extending OpenVINO:tm: Python API
+
+###### Before start: Project's naming conventions
+General guide:
```

Comment:
I'm not sure if that true but looks like naming convention is like [PEP8 naming convention](https://peps.python.org/pep-0008/#naming-conventions)?

Maybe mention it here and write what is a different from this standard?

In PEP there is also mention about styles:
* UPPERCASE
* UPPER_CASE_WITH_UNDERSCORES
etc.

In what cases we can used here or maybe is not allowed at all.

Reply:
I forgot that PEP8 actually mentioned it, thank you for a pointer!